src/app/page.tsx

```diff
@@ -82,10 +137,12 @@ export default function Home() {
             <div className="w-1/4">Graph</div>
           </div>
         </div>
-        <div className="mt-4 h-[672px] space-y-2">
-          {marketData.map((coin, index) => (
-            <CoinDetails key={coin.id} data={coin} spot={index} />
-          ))}
+        <div className="mt-4 space-y-2 flex justify-center items-center flex-col w-full">
+          {marketData.map((coin, index) => {
+            if (index >= detailsValue && index <= detailsValue + 9) {
+              return <CoinDetails key={coin.id} data={coin} spot={index} />;
+            }
+          })}
```

Comment:
this is not how you use map, you need to filter then map, or map, but add an else where you return null

Reply:
Should be good now